### ISSUE TITLE
Accordion accessibility bugs/investigations

### ISSUE THREAD

**Issue by LFDanLu** [adobe/react-spectrum#1726]
# 🐛 Bug Report 

From the latest testing session:
- Focus ring indicator appears on Android when using TalkBack to expand/collapse the Accordion rows
- Disabled items aren't announced as disabled when using TalkBack/VoiceOver. Disabled items also don't announce their expanded/collapsed state

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| react-spectrum   |
| Browser          |
| Operating System |

**Comment by intergalacticspacehighway** [comment]
@LFDanLu I quickly analyzed this. 
1. Focus ring - This happens as isFocusVisible becomes true due to some reason when Talkback is enabled and a button is pressed using double tap. The same happens with Tabs.

2. Talkback is picking heading tag's semantics as Button is wrapped using h3. Not sure what should be a better approach here for this fix. Changing role="presentation" of H3 or moving it into the button fixes it (this creates some style issues, but I think that would be an easy fix).

**Comment by devongovett** [comment]
First point appears to be fixed. Disabled point is still an issue.

**Comment by dannify** [comment]
Fixed in latest version of Accordion
